pkgs/development/python-modules/tensorflow/default.nix

```
@@ -236,6 +236,9 @@ let
     # arbitrarily set to the current latest bazel version, overly careful
     TF_IGNORE_MAX_BAZEL_VERSION = true;
 
+    # aarch64-darwin needs this env set
+    LIBTOOL = "${cctools}/bin/libtool";
```

```suggestion
    LIBTOOL = lib.optionalString stdenv.isDarwin "${cctools}/bin/libtool";
```
cctools are mac-specific